sharding/indexHashedNodesCoordinator.go

```diff
@@ -480,6 +525,25 @@ func (ihgs *indexHashedNodesCoordinator) EpochStartPrepare(metaHeader data.Heade
 		return
 	}
 
+	allValidators := make([]Validator, 0)
+
+	for _, shardValidators := range nodesConfig.eligibleMap {
+		allValidators = append(allValidators, shardValidators...)
+	}
+
+	for _, shardValidators := range nodesConfig.waitingMap {
+		allValidators = append(allValidators, shardValidators...)
+	}
+
+	sort.Slice(allValidators, func(i, j int) bool {
+		return bytes.Compare(allValidators[i].PubKey(), allValidators[j].PubKey()) < 0
+	})
+
+	leaving, err := ihgs.nodesPerShardSetter.ComputeLeaving(allValidators)
+	if err != nil {
+		log.Error("compute leaving failed", "error", err.Error())
```

Comment:
shouldn't you stop processing in case of error ? maybe return error in the interface method as well.

Reply:
ComputeLeaving does not return an error any more.